react-accordion/my-app/src/TopicCard.tsx

```diff
@@ -0,0 +1,22 @@
+type TopicCard = {
```

Comment:
This is essentially the same as TopicProp in AccordionWrap.tsx. It would be much better to have it only once, where you export it, and then import it in the other places you need it.

You can import types in exactly the same way you import functions.